example/spec/parallel/orca/OrcaBackupAndRecoverySpec.js

```diff
@@ -222,6 +225,7 @@ describe('The S3 Ingest Granules workflow', () => {
         { granuleId },
         'completed'
       );
+      await waitForGranuleRecordsInList(config.stackName, [granuleId]);
```

Comment:
Was this part of this ticket or a quick fix from something else?

Reply:
I had a test failure, and it appears that the record is not in es yet, this is just to make sure the record has been added to es.  It's not part of the ticket, just a fix for race condition.